trade-finance-manager-api/src/v1/deals/routes.js

```diff
@@ -76,6 +77,10 @@ dealsAuthRouter
   .get(validation.dealIdValidation, handleExpressValidatorResult, dealController.getDeal)
   .put(validation.dealIdValidation, handleExpressValidatorResult, dealController.updateDeal);
 
+dealsAuthRouter
+  .route('/deals/:dealId/cancellation')
+  .put(validation.dealIdValidation, handleExpressValidatorResult, dealCancellationController.updateDealCancellation);
```

Comment:
Do we want zod schema validation here as well? It'd be a new pattern but we could possibly move some stuff from dtfs-central to libs common (perhaps in a different subtask)

Reply:
I think that would be nice - but might cause this PR to take longer to get through. Happy for this to go as another slightly lower priority subtask?

Reply:
Sounds good - feel free to resolve this comment

Reply:
Added it as a subtask to the ticket👍